src/math/average_median.jl

```diff
@@ -1,5 +1,3 @@
-include("ceil_floor.jl")
-
 """
     median(nums)
 
```

Comment:
I think these should use `##` instead (same until line 15).

Reply:
I am not 100% sure of how it will look with ## ....
We can try to make one a double # and see how it looks :p

One # makes it a kind of bold paragraph title